.github/workflows/release.yml

```diff
@@ -31,5 +31,35 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Release iOS
+      - name: Release iOS, MacOS
         run: ./gradlew bintrayUpload -Ptarget=all_macos_hosted -Pbintray_user=${{ secrets.BINTRAY_USER }} -Pbintray_key=${{ secrets.BINTRAY_KEY }}
+  linux-host-check:
+    name: Check on Ubuntu
+    runs-on: ubuntu-latest
+    needs: linux-host-publish
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install dependencies
+        run: sudo apt-get install nodejs chromium-browser
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Check JVM, Android, JS, Linux
+        run: ./gradlew :tools:check-publication:build -Ptarget=all_linux_hosted -Pbintray_user=${{ secrets.BINTRAY_USER }} -Pbintray_key=${{ secrets.BINTRAY_KEY }} -Dcheck_publication=true
```

Comment:
Can we use project property instead of system property?
You can use `findProperty` or `hasProperty` method.

Reply:
I tried, these functions are not resolved in settings.gradle, for some reason.

Reply:
Try the following please:
```kotlin
val check_publication: String? by settings
if (check_publication == "true") {
...
}
```

Reply:
Issue: https://github.com/gradle/kotlin-dsl-samples/issues/635